deploy/crds/org.eclipse.che_checlusterbackups_crd.yaml

```diff
@@ -0,0 +1,214 @@
+                which one to use. Allowed values are fields names form BackupServers
+                struct.
+              type: string
+            servers:
+              description: List of backup servers. Usually only one is used. In case
+                of several available, ServerType should contain server to use.
+              properties:
+                awss3:
+                  description: Amazon S3 or alternatives
+                  properties:
+                    awsAccessKeyId:
+                      description: Content of AWS_ACCESS_KEY_ID environment variable
+                      type: string
+                    awsAccessKeySecretRef:
+                      description: Reference to secret that contains awsAccessKeyId
+                        and awsSecretAccessKey fields
+                      type: string
+                    awsSecretAccessKey:
+                      description: Content of AWS_SECRET_ACCESS_KEY environment variable
+                      type: string
+                    hostname:
+                      description: Server hostname, defaults to 's3.amazonaws.com'.
+                        Might be customized in case of alternative server.
+                      type: string
+                    port:
+                      description: Backup server port. Usually default value is used.
+                        Might be customized in case of alternative server.
+                      type: integer
+                    protocol:
+                      description: Protocol to use when connection to the server.
+                        Might be customized in case of alternative server.
+                      type: string
+                    repo:
+                      description: Bucket name and repository, e.g. bucket/repo
+                      type: string
+                    repoPassword:
+                      description: Restic repository password
+                      properties:
+                        password:
+                          description: Password for restic repository
+                          type: string
+                        passwordSecretRef:
+                          description: Secret with 'repo-password' filed
+                          type: string
+                      type: object
+                  type: object
+                internal:
+                  description: Rest server within the cluster. The server and configuration
+                    are created by operator when AutoconfigureRestBackupServer is
+                    true
+                  properties:
+                    credentialsSecretRef:
+                      description: Secret that contains username and password fields
+                      type: string
+                    hostname:
+                      description: Backup server host
+                      type: string
+                    password:
+                      description: Password to authenticate the user
```

Comment:
it's just an alternative to credentialsSecretRef, right? If so - let's make it clear in JSON Schema